test/fixtures/schema.sql

```diff
@@ -2041,9 +2041,12 @@ returns setof v2.parents as $$
   select * from v2.parents where id < $1;
 $$ language sql;
 
--- Only used for manually testing creating prepared statements
-create view prepared_statements as
-select * from pg_catalog.pg_prepared_statements;
+-- Used to test if prepared statements are used
+create function uses_prepared_statements() returns bool as $$
+    select c > 0 from (
+      select count(name) as c from pg_catalog.pg_prepared_statements
+    ) ps;
```

Comment:
I think this can be shortened to:
```suggestion
  select count(name) > 0 from pg_catalog.pg_prepared_statements;
```